core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -196,7 +196,7 @@ protected void doReadFrom(StreamInput in) throws IOException {
 
     @Override
     public final void writeTo(StreamOutput out) throws IOException {
-        out.writeString(name);    // NORELEASE remote writing the name? it is automatically handled with writeNamedWriteable
```

Comment:
I'm removing this because it was wrong - that `name` is the name that the user gave to the aggregation not the name of the writeable.